### DISCUSSION TITLE
2023/11/13/use-cloudflare-speed-up-github-pages/index

### DISCUSSION THREAD

**Discussion by giscus** [sumy7/sumy7.github.io#9] [Announcements]
# 2023/11/13/use-cloudflare-speed-up-github-pages/index

SumyGG's Blog

https://sumygg.com/2023/11/13/use-cloudflare-speed-up-github-pages/index.html

**Comment by zl-Diamon** [comment]
你好，我这边配置了以后，域名都ping不同来着，可能是什么原因呢，我也是刚刚转到cloudflare，

**Comment by sumy7** [comment]
可以查看一下域名的DNS解析，是否切换到cloudflare的服务器上面了。

**Comment by DopplerXD** [comment]
成功了，点赞！
